javascript/javascript3/week3/Shopping_cart/main.js

```diff
@@ -0,0 +1,113 @@
+const body = document.querySelector("body");
+
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
```

Comment:
Just for a future improvement you could try using an existing api to do the conversion. For instance this:
https://free.currencyconverterapi.com/

Reply:
Thanks a lot Dimitris for your review. It's helpfull. Have a nice week.